docs/examples/Themis-server/go/ssession_server.go

```diff
@@ -81,53 +81,53 @@ func clientService(client *session.SecureSession, ch chan []byte, finCh chan int
 	finCh <- 1
 }
 
-func createSecureSession(inputBuffer *bufio.Reader) (*session.SecureSession, string) {
+func createSecureSession(inputBuffer *bufio.Reader) (*session.SecureSession, string, error) {
 	fmt.Println("Type your settings from https://themis.cossacklabs.com/interactive-simulator/setup/")
 
 	fmt.Println("JSON endpoint: ")
 	endpoint, err := inputBuffer.ReadString('\n')
 	if err != nil {
-		fmt.Println("Failed to read endpoint URL:", err)
-		return nil, ""
+		err = fmt.Errorf("Failed to read endpoint URL:", err)
```

Comment:
you use `fmt.Errorf` and don't use any pattern for formatting: https://play.golang.org/p/3z5WE3eymGs . Use `URL: %s', err)`

Reply:
I have to agree, it's pretty dumb of me to introduce _new_ go vet warnings after getting rid of them :) Nice catch, thank you.